README.md

```diff
@@ -2,7 +2,103 @@
 
 [![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://invenia.github.io/FullNetworkSystems.jl/stable)
 [![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://invenia.github.io/FullNetworkSystems.jl/dev)
-[![Build Status](https://github.com/invenia/FullNetworkSystems.jl/actions/workflows/JuliaNightly.yml/badge.svg?branch=main)](https://github.com/invenia/FullNetworkSystems.jl/actions/workflows/JuliaNightly.yml?query=branch%3Amain)
+[![Build Status](https://github.com/invenia/FullNetworkSystems.jl/actions/workflows/CI.yml/badge.svg?branch=main)](https://github.com/invenia/FullNetworkSystems.jl/actions/workflows/CI.yml?query=branch%3Amain)
 [![Coverage](https://codecov.io/gh/invenia/FullNetworkSystems.jl/branch/main/graph/badge.svg)](https://codecov.io/gh/invenia/FullNetworkSystems.jl)
 [![Code Style: Blue](https://img.shields.io/badge/code%20style-blue-4495d1.svg)](https://github.com/invenia/BlueStyle)
 [![ColPrac: Contributor's Guide on Collaborative Practices for Community Packages](https://img.shields.io/badge/ColPrac-Contributor's%20Guide-blueviolet)](https://github.com/SciML/ColPrac)
+
+This package defines a set of types and data structures to represent an energy grid and the components within it.
+The types defined can be used to build Optimal Power Flow and associated optimisation problems e.g. unit commitment and economic dispatch.
+The data structures are designed so that the relevant data can be easily accessed and used to formulate these optimisation problems.
+
+## Example Modelling Problem
+
+The following is an example of how the data stored in a `SystemRT` object can be used to build a JuMP model.
+The objective of the model is to solve a simple energy balance problem, where a set of loads (demands) need to be met by generation (supply), for the minimum possible cost.
+Assume we have an instance of a system (see [`SystemRT`](@ref) for details of all the components in a `SystemRT`).
+
+```julia
+using AxisKeys
+using FullNetworkSystems
+using HiGHS
+using JuMP
+
+system = SystemRT(components...)
+
+model = Model()
+units = keys(get_generators(system))
+datetimes = get_datetimes(system)
+
+@variable(model, generation[u in units, t in datetimes] >= 0)
+
+load = get_loads(system)
+@constraint(
+    model,
+    energy_balance[t in datetimes],
+    sum(model[:generation][u, t] for u in units) == sum(load(l, t) for l in axiskeys(load, 1))
```

Comment:
where does `load` come from? or is this jump magic introducing it?

Reply:
`load` is defined above from `get_load(system)`.  It is a `KeyedArray` so uses `()` for indexing by the names of the keys within the `@constraint` definition.